### PR TITLE
backport connectors condensing

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Backporting based on https://github.com/camunda/camunda-platform-docs/pull/2656

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [x] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Review by sbuettner — DISMISSED** [review]

File: static/.htaccess [hunk 1]
```diff
@@ -9,27 +9,43 @@ Options -Indexes -MultiViews
...
 RewriteRule ^docs/components/connectors/out-of-the-box-connectors/amazon-sqs-inbound/(.*)$ /docs/components/connectors/out-of-the-box-connectors/amazon-sqs/$1 [R=301,L]
+RewriteRule ^docs/8.2/components/connectors/out-of-the-box-connectors/amazon-sqs-inbound/(.*)$ /docs/8.2/components/connectors/out-of-the-box-connectors/amazon-sqs/$1 [R=301,L]
+RewriteRule ^docs/8.1/components/connectors/out-of-the-box-connectors/amazon-sqs-inbound/(.*)$ /docs/8.1/components/connectors/out-of-the-box-connectors/amazon-sqs/$1 [R=301,L]
 
-RewriteRule ^docs/next/components/connectors/out-of-the-box-connectors/amazon-eventbridge-inbound/(.*)$ /docs/next/components/connectors/out-of-the-box-connectors/amazon-eventbridge/$1 [R=301,L]
-RewriteRule ^docs/components/connectors/out-of-the-box-connectors/amazon-eventbridge-inbound/(.*)$ /docs/components/connectors/out-of-the-box-connectors/amazon-eventbridge/$1 [R=301,L]
+RewriteRule ^docs/next/components/connectors/out-of-the-box-connectors/amazon-eventbridge-webhook/(.*)$ /docs/next/components/connectors/out-of-the-box-connectors/amazon-eventbridge/$1 [R=301,L]
```
Comment (pepopowitz) [inline static/.htaccess:20]:
```suggestion
```

Per [this comment/decision](https://github.com/camunda/camunda-platform-docs/issues/1397#issuecomment-1677542780), we shouldn't worry about redirects for `/next` docs anymore.

**Review by pepopowitz — CHANGES_REQUESTED** [review]
One tiny change requested!

File: static/.htaccess (hunk 1 above)
Comment (christinaausley) [reply static/.htaccess:20]: Wahoooooo!!

**Review by pepopowitz — APPROVED** [review]
:shipit:

**Pull request merged**